bindata/network/ovn-kubernetes/common/alert-rules.yaml

```diff
@@ -26,3 +26,15 @@ spec:
       for: 20m
       labels:
         severity: warning
+    - alert: NodeWithOVNContollerDisconnectedFromSBDB
+      annotations:
+        summary: All nodes should be having stable connections between their ovn-controller and SBDB, {{"{{"}} $labels.node {{"}}"}} does not.
+        # runbook_url: TODO by https://issues.redhat.com/browse/SDN-3444
+        description: |
+          Networking is degraded on nodes that do not have a healthy ovn-controller <-> SBDB connection. We will fire an alert
+          if ovn-controller to SBDB connection has been down for >= 10 minutes at a stretch.
+      expr: |
+        ovn_controller_southbound_database_connected == 0
```

Comment:
@martinkennelly : I guess we should be good here, since the metric is scraped once every 2 mins. So there should be 5 scrapes all returning 0 for us to fire alert...